modules/javafx.graphics/src/main/native-prism-d3d/D3DPipeline.cc

```diff
@@ -237,12 +229,12 @@ void fillOsVersionInformation(JNIEnv *env, jobject object, jclass clazz) {
     }
 }
 
-inline IDirect3D9* addRef(IDirect3D9* i) {
+inline IDirect3D9Ex* addRef(IDirect3D9Ex* i) {
     i->AddRef();
     return i;
 }
 
-int getMaxSampleSupport(IDirect3D9 *d3d9, UINT adapter) {
+int getMaxSampleSupport(IDirect3D9Ex *d3d9, UINT adapter) {
```

Comment:
Minor: In some cases you also change the name of the variable to add the "Ex" suffix., like in
```cpp
D3DContext::D3DContext(IDirect3D9Ex *pd3dEx, UINT adapter)
                                         ^
```
Here and In `PiplineManager.h` it's left as `IDirect3D9Ex * pd3d9;` without "Ex".
I don't mind it either way (I would probably not bother changing them myself), but perhaps we should remain consistent.

Reply:
You're right. Most places use it as `pd3d9` so I'll leave it this way (remove the `Ex` suffix from other spots).